src/common/cli_wrapper.cpp

```diff
@@ -4,7 +4,8 @@
 namespace marian {
 namespace cli {
 
-uint16_t guess_terminal_width(uint16_t max_width, uint16_t default_width) {
+/*
+static uint16_t guess_terminal_width(uint16_t max_width, uint16_t default_width) {
```

Comment:
This caused a warning "unused local function was deleted". My first idea was to remove it, but in the code below there is a TODO to compute the terminal size with this function. So I just commented it for now.

Reply:
(Un-resolving only so that this does not get missed. Please just Resolve it once you saw this, no need for action.)

In such cases, I found it is sometimes easier to change the function definition to 'static inline'. I use that trick for static functions headers. Probably not here though.